src/transport/HighPressureGasTransport.cpp

```diff
@@ -327,167 +323,579 @@ double HighPressureGasTransport::viscosity()
+            // double summation into this variable, and then later square the result.
+            double MW_ij =  (2*MW_i[i]*MW_i[j]) / (MW_i[i] + MW_i[j]); // Equation 9-5.40
+            params.MW_mix += molefracs[i]*molefracs[j]*epsilon_over_k_ij*pow(sigma_ij,2.0)*sqrt(MW_ij); // Equation 9-5.28 (double summation only)
+
+            double acentric_factor_ij = 0.5*(acentric_factor_i[i] + acentric_factor_i[j]); // Equation 9-5.36
+            params.acentric_factor_mix += molefracs[i]*molefracs[j]*acentric_factor_ij*pow(sigma_ij,3.0); // Equation 9-5.29
+
+            // The base class' dipole moment values are in the SI units, so we need to convert to
+            // Debye units.
+            double SI_to_Debye = 1.0 / 3.335e-30; // Conversion factor from C*m to Debye
+            double dipole_ii = m_dipole(i,i)*SI_to_Debye;
+            double dipole_jj = m_dipole(j,j)*SI_to_Debye;
+            params.mu_mix += molefracs[i]*molefracs[j]*pow(dipole_ii,2.0)*pow(dipole_jj,2.0)/pow(sigma_ij,3.0); // Equation 9-5.30
+
+            // Using equation 9-5.31
+            double kappa_ij = sqrt(kappa_i[i]*kappa_i[j]); // Equation 9-5.39
+            params.kappa_mix += molefracs[i]*molefracs[j]*kappa_ij; // Equation 9-5.31
+        }
     }
 
-    double P_corr_1 = DP_Rt_lookup[Pr_i]*(1.0 - A_ij_lookup[Pr_i]
-        *pow(Tr,-B_ij_lookup[Pr_i]))*(1-C_ij_lookup[Pr_i]
-        *pow(Tr,-E_ij_lookup[Pr_i]));
-    double P_corr_2 = DP_Rt_lookup[Pr_i+1]*(1.0 - A_ij_lookup[Pr_i+1]
-        *pow(Tr,-B_ij_lookup[Pr_i+1]))*(1-C_ij_lookup[Pr_i+1]
-        *pow(Tr,-E_ij_lookup[Pr_i+1]));
-    return P_corr_1*(1.0-frac) + P_corr_2*frac;
+    // Finalize the expressions for the mixture values of the parameters
+    params.sigma_mix = pow(params.sigma_mix, 1.0/3.0); // Equation 9-5.25 computed the cube of sigma_mix
+    params.epsilon_over_k_mix /= pow(params.sigma_mix,3.0);
+
+    // The MW_mix was only the numerator inside the brackets of equation 9-5.28
+    params.MW_mix = pow(params.MW_mix/(params.epsilon_over_k_mix*pow(params.sigma_mix,2.0)), 2.0);
+
+    params.acentric_factor_mix /= pow(params.sigma_mix, 3.0);
+
+    params.mu_mix = pow(pow(params.sigma_mix, 3.0)*params.mu_mix, 1.0/4.0); // Equation 9-5.30 computed the 4th power of mu_mix
+
+    // Tc_mix is computed using equation 9-5.44.
+    params.Tc_mix = 1.2593*params.epsilon_over_k_mix;
+
+    // Vc_mix is computed using equation 9-5.43.
+    params.Vc_mix = pow(params.sigma_mix/0.809, 3.0);
+
+    // mu_r_mix is computed using equation 9-5.42.
+    params.mu_r_mix = 131.3*params.mu_mix/sqrt(params.Vc_mix*params.Tc_mix);
 }
 
+double neufeld_collision_integral(double T_star)
+{
+    double A = 1.16145;
+    double B = 0.14874;
+    double C = 0.52487;
+    double D = 0.77320;
+    double E = 2.16178;
+    double F = 2.43787;
+
+    double omega = A / pow(T_star, B) + C / exp(D*T_star) + E / exp(F*T_star);
+    return omega;
```

Comment:
Here and elsewhere, my recommendation is to save a line of code and just return the computed variable without storing it in a local variable:
```suggestion
    return A / pow(T_star, B) + C / exp(D*T_star) + E / exp(F*T_star);
```